pkg/build/api/v1/types.go

```diff
@@ -622,6 +622,19 @@ type BuildOutput struct {
 	// up the authentication for executing the Docker push to authentication
 	// enabled Docker Registry (or Docker Hub).
 	PushSecret *kapi.LocalObjectReference `json:"pushSecret,omitempty" protobuf:"bytes,2,opt,name=pushSecret"`
+
+	// labels define a list of labels that are applied to the resulting image. If there are
+	// multiple labels with the same name then the last one in the list is used.
```

Comment:
The idea was that the BuildDefaults admission plugin will prepend labels to the list making them effectively defaults and BuildOverrides will append them making them mandatory. Not really a problem though, this logic can be extended to remove the duplicate entry.